scripts/plot_performance_breakdown.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Given a benchmark result json that was generated with --sql_metrics, this script plots the cost of the different
+# steps (parsing, optimization, ...) as a normalized stacked bar chart.
+
+import json
+import matplotlib.pyplot as plt
+import pandas as pd
+import sys
+
+benchmarks = []
+
+if(len(sys.argv) != 2):
+    print("Usage: " + sys.argv[0] + " benchmark.json")
+    exit()
+
+with open(sys.argv[1]) as file:
+    data = json.load(file)
+
+for benchmark_json in data['benchmarks']:
+    benchmark = []
+    benchmark.append(benchmark_json['name'])
+
+    sum_parse_duration = 0.0
+    sum_sql_translation_duration = 0.0
+    sum_optimization_duration = 0.0
+    sum_lqp_translation_duration = 0.0
+    sum_plan_execution_duration = 0.0
+
+    for run in benchmark_json['successful_runs']:
+        if len(run['metrics']) == 0:
+            print("No metrics found. Did you run the benchmark with --sql_metrics?")
```

Comment:
`Exit("No metrics found. Did you run the benchmark with --sql_metrics?")` to return a non-success return code.